submissions/modupeAdeOnojobi/php-db/orders.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+require_once("connection.php");
+
+try{
+    $sql = "SELECT order_items.quantity, order_items.unit_prices, 
+    order_items.total_amount, orders.date, orders.id, customer_address.street, 
+    customer_address.city, customer_address.state, customer_address.country, products.product_name 
+    FROM order_items 
+    LEFT JOIN orders ON orders.id = order_items.order_id
+    LEFT JOIN customer_address on orders.customer_id = customer_address.id
+    LEFT JOIN products ON products.id = order_items.product_id 
+    WHERE orders.customer_id = order_items.id";
```

Comment:
Well done. But, each page reloads, this query is always be executed even when the cookie is set. So, I will advise you to write a separate function that handles this query, then call that function when the cookie is not set

Reply:
I have done that for both files. Kindly review my pull request